### PR TITLE
Responsive Admins and My Credits

### PR DESCRIPTION
**Pull request by cmdalbem**
#### What is the purpose of this pull request?
Makes our admins and My Credits responsive, with a minimal working version on Mobile.

#### What problem is this solving?
Our admins and My Credits are unusable on Mobile right now.

#### Screenshots or example usage

ambienteqa myvtex com__secure_account_orders_workspace cmdalbem uam true mobile 4

cmdalbem--ambienteqa myvtex com_admin_creditcontrol_invoices 2

cmdalbem--ambienteqa myvtex com_admin_creditcontrol_invoices 3

cmdalbem--ambienteqa myvtex com_admin_creditcontrol_invoices

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/StateButton.js [hunk 1]
```diff
@@ -32,7 +32,9 @@ class StateButton extends React.Component {
         style={
           isActive
             ? {
-                boxShadow: '0 2px #F71963',
+                borderBottom: '2px solid #F71963',
```
Comment (kevinch) [inline react/components/StateButton.js]: Is this color out of the Styleguide?

File: react/styles/react-table.css [hunk 2]
```diff
@@ -139,3 +139,33 @@ ReactTable .rt-thead .rt-td {
...
+
+@media screen and (max-width: 40em) { 
+  .ReactTable .rt-td {
+    flex: 1 !important;
+    width: 100% !important;
+    max-width: none !important;
+  }
+
+  .ReactTable .rt-thead.-header {
+    min-width: inherit !important;
```
Comment (kevinch) [inline react/styles/react-table.css]: Useful since `display` is set to `none`?

File: react/components/StateButton.js (hunk 1 above)
Comment (cmdalbem) [reply react/components/StateButton.js]: no, it's rebel pink, but it was already hardcoded like this.
the thing is that we're not using Styleguide's Tabs, we're using a custom one, but I didn't want to invest time in changing to use the Styleguide one now.

File: react/styles/react-table.css (hunk 2 above)
Comment (cmdalbem) [reply react/styles/react-table.css]: oops, good catch

**Pull request merged**